### DISCUSSION TITLE
Slider limit bypass

### DISCUSSION THREAD

**Discussion by Mar1Xd** [maxnut/GDmenu#294] [Q&A]
I accidentally put my slider way too much (on the other monitor) and now I can't get it back. I tried reinstaling the game , I tried putting my game on the second monitor, I tried reainstaling the mod and yet nothing worked. Can somebody help with this?

**Comment by XenonPhanes** [comment]
If you're talking about volume, In geode go to better pause, install that mod, go into a level and in the pause menu click on the volume percentage.
